frontend/javascripts/oxalis/model/helpers/nml_helpers.ts

```diff
@@ -729,11 +714,14 @@ export function parseNml(nmlString: string): Promise<{
                 Math.trunc(_parseFloat(attr, "y")),
                 Math.trunc(_parseFloat(attr, "z")),
               ] as Vector3,
-              additionalCoordinates: _parseJSON(
-                attr,
-                "additionalCoordinates",
-                [],
-              ) as AdditionalCoordinate[],
+              // Parse additional coordinates, like additionalCoordinate-t="10"
+              additionalCoordinates: Object.keys(attr)
```

Comment:
The String's `startsWith` method in combination with `slice(len("additionalCoordinate-")` could be a more readable replacement here, but also fine as-is.

Reply:
Since this needs the `additionalCoordinate` literal twice, I'm leaning against that :see_no_evil: